Documentation/getting-started-guide.md

```diff
@@ -60,6 +95,14 @@ acbuild write hello-0.0.1-linux-amd64.aci
 acbuild end
 ```
 
+### Enable notifications
```

Comment:
I will just add a line with a reference to https://github.com/coreos/rkt/blob/master/Documentation/using-rkt-with-systemd.md